setup.py

```diff
@@ -44,6 +44,8 @@
     ],
     "optional": [
         "pymupdf>=1.22.0",  # logbook for conversion from pdf to png
+        "qtpy>=2.3.1",  # putting plots in windows
+        "pyside2>=5.15.0.0",  # putting plots in windows
```

Comment:
Why do we force this? I thought the point of `qtpy` was that it would pick up any bindings on the local machine and make do with that for us. Shouldn't we just have `qtpy` as a dependency?

Reply:
Also, `pyside2` is under `LGPL` and I'm still not convinced the optional dependency trick really holds, given we're `MIT` licensed and want to stay that way.

Reply:
Also (yeah I know), `pyside2` provides bindings to `Qt5` which [has already reached end of life](https://endoflife.date/qt). So if we're forcing bindings, shouldn't we go for `pyside6` then?

Reply:
Alright for LGPL. Regarding the following:
>I would be very annoyed if I would try to run the code with qtpy and it would still complain about missing dependencies that are not mentioned anywhere

I feel like going through `qtpy` means letting it handle the bindings and leaving the responsibility of having them installed to the user, while also being very clear about this in the docs.

As it's optional though it's not a huge deal for sure. But then having `pyside6` (cf [d5c0fb8](https://github.com/pylhc/omc3/pull/416/commits/d5c0fb8504207f2b2c4fb2d4f9ef0436e4ae4081)) specified feels like we're "forcing" (big airquotes here) some bindings on the user.

Reply:
To be clear, I'm not a fan but I don't consider this blocking.